src/Report.php

```diff
@@ -285,6 +287,20 @@ public function getName()
     }
 
     /**
+     * Get the short name.
+     *
+     * @return string
+     */
+    public function getShortName()
+    {
+        try {
+            return (new ReflectionClass($this->name))->getShortName();
```

Comment:
If an error had a name of `Symfony\Component\HttpKernel\Exception\TooManyRequestsHttpException`, then this will give us `TooManyRequestsHttpException`, which is much better for a breadcrumb. If this ends up longer than 30 characters, that's fine too since this will get trimmed later on.